Cargo.toml

```
@@ -78,7 +78,7 @@ default-features = false
 features = ["parsing"]
 
 [dependencies.clap]
-version = "4.0.32"
+version = "4.1.8"
```

I think we could potentially put 
```suggestion
version = "4"
```
here in order to avoid having to update this manually in the future (unless there's a new major version).

---

In general it is better to put a lower bound on minors versions, otherwise cargo can resolve versions in the wrong way.

More details and further discussion here: https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277

---

...and dependabot does the bumps for us anyway, right? In this case I think we asked dependabot to stop since bumping required manual work. From what I understand, us doing the bump manuwally now will make dependabot start bumping clap for us again.

---

Ok, thanks for the explanation. I reverted the change